packages/manager/modules/pci/src/projects/project/translations/Messages_fr_FR.json

```diff
@@ -13,20 +13,23 @@
   "pci_projects_project_is_new": "Nouveau",
   "pci_projects_project_documentation": "Documentation",
   "pci_projects_project_community": "Communauté",
-  "pci_projects_project_documentation_public_cloud_essential_information": "L’essentiel pour commencer avec Public Cloud",
-  "pci_projects_project_documentation_public_cloud_interface": "Se familiariser avec l’interface Public Cloud",
-  "pci_projects_project_documentation_get_started_with_a_public_cloud_instance": "Gérer vos instances",
-  "pci_projects_project_documentation_information_on_cloud_billing_options": "Comprendre et gérer sa facturation",
-  "pci_projects_project_documentation_create_user_access_to_horizon": "Create user access to Horizon",
-  "pci_projects_project_documentation_boot_your_first_cloud_server_in_3_minutes": "Boot your first cloud server in 3 minutes",
-  "pci_projects_project_documentation_prepare_the_environment_for_using_the_open_stack_api": "Prepare the environment for using the OpenStack API",
-  "pci_projects_project_documentation_create_and_configure_and_additional_disk_on_an_instance": "Create and configure and additional disk on an instance",
-  "pci_projects_project_documentation_see_all_public_cloud_guides": "Voir tous les guides Public Cloud",
-  "pci_projects_project_documentation_how_to_create_a_horizon_user": "How to Create a Horizon User",
-  "pci_projects_project_documentation_how_to_deploy_a_public_cloud_instance": "How to Deploy a Public Cloud Instance",
-  "pci_projects_project_documentation_getting_started_with_block_storage": "Getting Started with Block Storage",
-  "pci_projects_project_community_roadmap": "Découvrez et participez à la roadmap de nos produits",
-  "pci_projects_project_community_discord": "Discutez avec la communauté sur Discord",
+  "pci_projects_project_billing": "Facturation",
+  "pci_projects_project_billing_voucher_credit": "Code promotion : {{ voucher }}",
+  "pci_projects_project_billing_active_credit_balance": "Valide jusqu'au {{ expirationDate }}",
+  "pci_projects_project_documentation_getting_started_term": "Getting Started",
```

Comment:
**question:** Do we need translation for this?

Reply:
Yes, we do.

Reply:
This is the label indicated by the UI specification.

Reply:
So, If I understand, other languages will have this label translated, but french not?

Reply:
Sorry Guys, I misunderstood the implication of your question. 
So there is no need for translation for this term.
As you can see in the JIRA ticket : https://projects.dsi.ovh/browse/MANAGER-9862 ~ FREE TRIAL 3.PNG : the label of this entry is "Getting Started".